README.md

```diff
@@ -202,6 +202,7 @@ Configuration reference
   * clone_from_vm - Source vm to clone. Mutually exclusive with ovf_source option.
   * ovf_source - ovf files or URLs to use as a source. Mutually exclusive with clone_from_vm option.
   * disk_store - Required - esxi Disk Store where guest vm will be created.
+  * host_iso - Optional - Attaching an iso file on the ESXi host. - e.g. "datastore1/iso/centos.iso"
```

Comment:
Could you rename this resource to "cdrom_datastore_iso".  It will match (closer) the web gui description so it will be a bit more clear.